swagger/public_v1/api_spec.json

```diff
@@ -1240,6 +1240,45 @@
         "example": "",
         "description": "This schema is used to search within collections to return more specific results.",
         "properties": {
+          "can_sponsor_skilled_worker_visa": {
+            "description": "Only return providers that can sponsor skilled worker visa.",
```

Comment:
```suggestion
            "description": "Only return providers that can sponsor a Skilled Worker visa.",
```

Reply:
Done in https://github.com/DFE-Digital/publish-teacher-training/pull/3150/commits/fc81aef3dbef15ef4a799576b67adbd1d9f472b7